scripts/download_sfs_metadata.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import argparse
+import json
+import csv
+
+def get_metadata_from_id3c(url, username, password, output):
+    """
+    GET metadata from the ID3C server and print data stream to *output*.
+    """
+    r = requests.get(url, auth=(username,password), stream=True)
+    stream = r.iter_lines()
+
+    # Save first line to use for writing TSV header
+    first_stream = json.loads(next(stream))
+
+    with open(output, 'w+') as tsv_file:
+        tsv_writer = csv.writer(tsv_file, delimiter='\t')
+
+        tsv_writer.writerow(first_stream.keys())
+        tsv_writer.writerow(first_stream.values())
+
+        for line in stream:
+            if line:
+                tsv_writer.writerow(json.loads(line).values())
```

Comment:
One way to avoid the need to save the first line (which assumes there's always at least one record) is to do something like the following instead:

```python
for i, record in enumerate(map(json.loads, stream)):
    # Write the header
    if i == 1:
        tsv_writer.writerow(record.keys())

    tsv_writer.writerow(record.values())
```

Relatedly, we should confirm that `json_to_record` in Postgres is going to return stable JSON key ordering across invocations, otherwise the order of keys and values will be messed up in this TSV.

Ultimately, it would nice to teach augur to read NDJSON files directly, which should be a pretty small change. :-)